src/test/regress/bin/create_test.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+import sys
+import random
+import os
+
+if len(sys.argv) != 2:
+    raise Exception("Expected the name of the new test as an argument")
```

Comment:
maybe we can be even more explicit and give an example command so that people know what to write directly.